glass/core/array.py

```diff
@@ -2,19 +2,21 @@
 
 from __future__ import annotations
 
-import typing
 from functools import partial
+from typing import TYPE_CHECKING
 
 import numpy as np
-import numpy.typing as npt
+from typing_extensions import Unpack
```

Comment:
Move into TYPE_CHECKING block and remove dependency

Reply:
Good idea, have raised https://github.com/glass-dev/glass/pull/517